### PR TITLE
Use langver=latest for scripting

### PR DESCRIPTION
**Pull request by jcouv**
The Interactive Window was using langver=default. This PR makes it use langver=latest.

Fixes https://github.com/dotnet/roslyn/issues/21327

### CODE REVIEW

File: src/Scripting/VisualBasicTest/CommandLineRunnerTests.vb [hunk 1]
```diff
@@ -186,6 +186,33 @@ Type ""#help"" for more information.
...
+            AssertEx.AssertEqualToleratingWhitespaceDifferences(
+"Microsoft (R) Visual Basic Interactive Compiler version " + s_compilerVersion + "
+Copyright (C) Microsoft Corporation. All rights reserved.
+Type ""#help"" for more information.
+>
+> Dim a = 1
+> Dim t = (a, 2)
+> System.Console.Write(t.a)
+«Red»
+Public member 'a' on type 'ValueTuple(Of Object,Integer)' not found.
```
Comment (jcouv) [inline src/Scripting/VisualBasicTest/CommandLineRunnerTests.vb]: @tmat I was surprised by the behavior of VB scripting with tuple names. I'll investigate before merging this PR, but I wondered if the problem might be obvious to you. Any clues?

Comment (tmat) [inline src/Scripting/VisualBasicTest/CommandLineRunnerTests.vb]: VB interactive isn't complete. There is a lot of missing pieces currently. I'd not bother writing test for VB.

**Comment by tmat** [comment]
We'll also need to change this in the IDE (along with the binding flag to ignore corlib duplicate).

**Comment by jcouv** [comment]
Deploying from command-line worked. Verified the fix in dev hive. Thanks

image

**Comment by tmat** [comment]
👍

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for review. Thanks

**Comment by jcouv** [comment]
test windows_debug_vs-integration_prtest please

**Comment by jcouv** [comment]
test windows_release_vs-integration_prtest please

**Comment by jcouv** [comment]
test windows_debug_vs-integration_prtest please

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for review. Thanks

File: src/Scripting/CSharpTest/CommandLineRunnerTests.cs [hunk 2]
```diff
@@ -947,5 +947,48 @@ public void PreservingDeclarationsOnException()
...
+Type ""#help"" for more information.
+> int i = default;
+> Print(i);
+0
+> ", runner.Console.Out.ToString());
+        }
+
+        [Fact]
+        [WorkItem(21327, "https://github.com/dotnet/roslyn/issues/21327")]
+        public void InferredTupleNames()
```
Comment (cston) [inline src/Scripting/CSharpTest/CommandLineRunnerTests.cs]: Consider adding an equivalent test for VB.

Comment (jcouv) [inline src/Scripting/CSharpTest/CommandLineRunnerTests.cs]: I had one, but removed it on advice of Tomas. VB scripting is not sufficiently functional at this point.
The test was producing a missing type error for ValueTuple type from some CompilerServices helper type.

**Comment by cston** [comment]
LGTM

**Comment by filipw** [comment]
note that this is already supported in Omnisharp and VS Code too (scripting is using latest lang version) 😄

**Pull request merged**